react/week2/todoapp/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import "./index.css";
+import ToDoList from './ToDoList'
+import Timer from './Timer.js'
+
+function App() {
+  const[inputlist,setInputList]=useState();
+  const[items,setItems]= useState([]);
+
+  const itemEvent=(event)=>{
+    setInputList(event.target.value);
+
+  }
+
+  const lisOfItems=()=>{
+    setItems((oldItem)=>{
+      return[...oldItem,inputlist];
+      
+    });
+    setInputList('');
+  }
+
+  const deleteItems=(id)=>{
+  
+    
+    setItems((oldItem)=>{
+      return oldItem.filter((arrElement, index)=>{
+              return index!==id;
+            });
+          });
+  }
+
+  return (
+    <>
+      
+      <div >
+        <br />
```

Comment:
It is not semantically correct to create space between elements by using <br>. For that you can use padding/margin in CSS. It's because HTML is parsed by search engines and thus it tells a story about the content of the page. If you want to build websites that are optimized for search results, you have to structure your HTML with the appropriate HTML tags.